doc/release/RELEASE-NOTES.md

```diff
@@ -31,6 +31,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 *
 
 ### What's new
+* [OSDEV-1045](https://opensupplyhub.atlassian.net/browse/OSDEV-1045) - The color of the facility panel for claimed facilities in the Merge moderation screens has been changed to green. Added flag `highlightBackground` for different screens to highlight panel only on the Dashboard Merge Facilities screen.
```

Comment:
Please update the description after implementing the changes according to the comments I provided below.